build.gradle

```diff
@@ -26,8 +26,11 @@ dependencies {
     implementation "org.springframework.boot:spring-boot-starter-data-jpa"
     implementation "org.springframework.boot:spring-boot-starter-security"
     implementation "org.springframework.boot:spring-boot-starter-oauth2-resource-server"
+//    implementation 'org.springframework.boot:spring-boot-starter-cache'
     implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.0.0'
+
     implementation "com.h2database:h2:2.1.210"
+//    implementation 'com.github.ben-manes.caffeine:caffeine'
```

Comment:
Test works w/out this dependency but it's needed  to use caffeine and it's cache configuration.